apps/cookbook/src/app/showcase/header-showcase/header-showcase.component.html

```diff
@@ -0,0 +1,96 @@
+<div class="example">
+  <div class="header-example-container">
+    <p>
+      Header is a component that lays out the header section of kirby-page. It is responsive and
+      will change the layout according to the device size. The header allows for a wide range of
+      customizability but you should follow the UX guidelines for header to provide the best
+      expereience.
+    </p>
+
+    <h2>Default</h2>
+
+    <cookbook-example-viewer [html]="defaultExample.template">
+      <div class="header-example">
+        <cookbook-header-example-default #defaultExample></cookbook-header-example-default>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Flag</h2>
+    <p>
+      A flag can be used to indicate a status. To use flag you simply add
+      <code>kirby-flag</code>
+      in the header and add the
+      <code>*kirbyHeaderFlag</code>
+      directive to it and that should position the flag correctly and manage spacing.
+    </p>
+    <cookbook-example-viewer [html]="flagExample.template">
+      <div class="header-example">
+        <cookbook-header-example-flag #flagExample></cookbook-header-example-flag>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Key-value</h2>
+    <p>
+      In some cases a number is the primary information on a page. When that's the case you should
+      use value adjust the header accordingly. To use value you simply add
+      <code>ng-container</code>
+      or another element with the value you want to display in the header and add the
+      <code>*kirbyHeaderValue</code>
+      directive to it and that should resize the title and display the value provided below.
+    </p>
+    <cookbook-example-viewer [html]="valueExample.template">
+      <div class="header-example">
+        <cookbook-header-example-value #valueExample></cookbook-header-example-value>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Actions</h2>
+    <p>
+      As an alternative to toolbar actions the header also allows you to insert actions in the
+      header itself. To insert actions simply add a
+      <code>ng-container</code>
+      or another element with the actions you want to display and add the
+      <code>*kirbyHeaderActions</code>
+      directive to it.
+    </p>
+    <cookbook-example-viewer [html]="actionsExample.template">
+      <div class="header-example">
+        <cookbook-header-example-actions #actionsExample></cookbook-header-example-actions>
+      </div>
+    </cookbook-example-viewer>
+  </div>
+
+  <div class="header-example-container">
+    <h2>Avatar</h2>
+    <p>
+      A header can be used together with a Avatar. You simply add a
+      <code>kirby-avatar</code>
+      in the header and add the
+      <code>*kirbyHeaderAvatar</code>
+      directive to it and that should position the avatar correctly and manage spacing.
```

Comment:
We should probably adjust this explanation after removing the `kirbyHeaderAvatar` directive:
```suggestion
      in the header which will position the avatar correctly and manage spacing.
```